sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/pom.xml

```diff
@@ -88,6 +88,17 @@
         </dependencies>
       </plugin>
     </plugins>
+
+    <pluginManagement>
```

Comment:
This is what I expect is the minimal configuration needed from the [parent pom.xml](https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/pom.xml#L164). I'm not familiar with archetype projects; @dhalperi, do you know how to test that this works?